smach_viewer/scripts/smach_viewer.py

```diff
@@ -165,10 +187,10 @@ def update_structure(self, msg):
 
     def _load_local_data(self, msg):
         """Unpack the user data"""
-        try:
-            local_data = pickle.loads(msg.local_data)
-        except KeyError:
-            local_data = pickle.loads(base64.b64decode(msg.local_data.encode('utf-8')))
+        if isinstance(msg.local_data, str):
+            local_data = pickle.loads(base64.b64decode(msg.local_data))
```

Comment:
this chage causes error in `melodic`.
```
[ERROR] [1658322156.379389]: bad callback: <bound method SmachViewerFrame._status_msg_update of <__main__.SmachViewerFrame; proxy of <Swig Object of type 'wxFrame *' at 0x55a2221c2ce0> >>
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/knorth55/ros/melodic/src/ros-visualization/executive_smach_visualization/smach_viewer/scripts/smach_viewer.py", line 879, in _status_msg_update
    if container.update_status(msg):
  File "/home/knorth55/ros/melodic/src/ros-visualization/executive_smach_visualization/smach_viewer/scripts/smach_viewer.py", line 217, in update_status
    self._local_data._data = self._load_local_data(msg)
  File "/home/knorth55/ros/melodic/src/ros-visualization/executive_smach_visualization/smach_viewer/scripts/smach_viewer.py", line 191, in _load_local_data
    local_data = pickle.loads(base64.b64decode(msg.local_data))
  File "/usr/lib/python2.7/base64.py", line 78, in b64decode
    raise TypeError(msg)
TypeError: Incorrect padding
```

Reply:
I haven't checked yet, but this may be caused by multibyte language...

Reply:
Actually it is not related to multibyte language.
we have same error with `roseus_smach/sample/state-machine-ros-sample.l`